### ISSUE TITLE
OS X Python 3.6.0a3 package was not uploaded

### ISSUE THREAD

**Issue by jjhelmus** [conda-forge/python-feedstock#63]
Looks like the Python 3.6.0a3 package in the `prerelease` branch was [built on Travis CI](https://travis-ci.org/conda-forge/python-feedstock/builds/144729252) but the upload command failed:

```
$ ./ci_support/upload_or_check_non_existence.py ./recipe conda-forge --channel=prerelease
Traceback (most recent call last):
  File "./ci_support/upload_or_check_non_existence.py", line 10, in <module>
    from binstar_client.utils import get_binstar
ImportError: No module named 'binstar_client'
```

**Comment by jakirkham** [comment]
Maybe try rerunning?  Almost everything else has been uploading ok. Though there have been some issues. They are not caused by an error like this. I don't think that script has changed since January.

Though I have seen some warnings about no `binstar_client`. It may be time to update that script. That being said, looks like we are planning on [removing it]( https://github.com/conda-forge/conda-smithy/pull/193/files#diff-62884a04ac0701bf0dc57453c360c940 ).

**Comment by jjhelmus** [comment]
Rerunning now.

**Comment by jjhelmus** [comment]
That fixed it, 3.6.0a3 is now available for OS X.  Thanks for the suggestion @jakirkham.
